test-projects/debug-log/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "worker",
+    "version": "1.0.0",
+    "dependencies": {
+        "@adobe/asset-compute-sdk": "^1.0.2"
```

Comment:
Why not use latest `asset-compute-sdk` (see https://github.com/adobe/asset-compute-sdk/blob/master/package.json#L5)?

Reply:
See #64